### DISCUSSION TITLE
Printing option

### DISCUSSION THREAD

**Discussion by hourrou** [freeplane/freeplane#1299] [Help]
Hi,

How can I center a map verticaly on a A4 landscape format please?

Thk you a lot,

Hrr

![image](https://github.com/freeplane/freeplane/assets/139434395/8e4c21b1-b236-43b7-bb4f-25295d982281)
![image](https://github.com/freeplane/freeplane/assets/139434395/e226fe34-33a1-4604-a6e2-95b198df924c)

**Comment by EdoFro** [comment]
maybe you should look at the printing properties of the printer and check there for this preference.

bye
